### PR TITLE
Updates biosilicified chitin material and armor

### PR DESCRIPTION
**Pull request by carlarctg**
#### Summary
Balance "Balanced and improved acidproof chitin"

#### Purpose of change

There's very few reasons to make stuff out of acidproof chitin. Despite the name, the armor is not actually acidproof, just slightly more resistant to it than chitin.

Adding insult to injury, for some reason it's also heavier, more encumbering, significantly less protective, and for some reason warmer. There's effectively no reason to ever use it, it's kind of... a jackass of all trades, except it was also made worse in every single possible way alongside that.

#### Describe the solution

Most acid chitin stats are now equal to chitin stats, with the following differences: It chips more, has 20-25% less M/R protection overall, has 20 acid resistance and 4 heat resistance. The acid resistance stat makes armor made out of it quite good at protecting you from acid, but not by any means immune, It still hurts a lot, but it lets you fight on the pool of acid for some small amount of time, which I think is the best of both worlds. Also it burns slower.

All acid chitin armor has had its thickness, encumbrance, and warmth made equal with chitin. There's already enough tradeoffs involved, it doesn't need even more.

Also, fixed an oversight in which chitin pet armor double dipped on chitin and steel resistances despite not even using the latter in its recipe, making armored pets nigh untouchable.

#### Describe alternatives you've considered

Making the armor fully acidproof - devs don't like it, armor has holes and fabric.
Some cool combination dualchitin armor - out of scope.
Making the armor BRITTLE rather than weaker - maybe.

#### Testing

Spawned myself in, examined all recipes, spawned biochitin armor, fought a giant ant, was concerned by heavy acid damage, realized I forgot to put my boots on. Tried again, values seem good.

#### Additional context

Has anyone ever used biosilsdclsdicdisf chitin armor???

Should its name be changed to acidproof because its so annoying to type

### CODE REVIEW

**Comment by fairyarmadillo** [comment]
I have some overhauls to how acid works coming later that might actually make this stuff a desirable alternative to normal chitin.

**Comment by carlarctg** [comment]
Feel free to change these stats when you PR your stuff. I'm looking forward to it but I won't rebalance this around a backburner project.

> I do feel it should have different physical resistances. Silicified means it's partly turned to glass, so it stands to reason it would be more rigid, so maybe +cut -bash?

Neat idea, sure.

**Comment by BlizzDA** [comment]
If the name has to be changed maybe add 'acid-resistant' to the armors' name or description. Since it isn't actually an acid-proof construction.

**Comment by PatrikLundell** [comment]
I feel the shoes should actually be acid proof, i.e. allow you to stand in a pool of acid without taking damage (it still wouldn't protect the rest of the body from acid spray). I don't see any point in using an acid proof material to make shoes and then leave holes in those shoes for acid (and water...) to leak in.

**Comment by fairyarmadillo** [comment]
> If the name has to be changed maybe add 'acid-resistant' to the armors' name or description. Since it isn't actually an acid-proof construction.

I don't think the name should be changed.

**Comment by carlarctg** [comment]
The fuck went wrong

**Comment by BlizzDA** [comment]
the new Treated Leather has a higher acid protection than normal leather. Could be used instead. Just remember to adjust the crafting recipe to take the 'treating' into account. 

Not entirely sure about how the chitin armor is supposed to look inregards to regular armor. Would there be any space for TL in the set's crafts, or is it more like scalemail?

**Comment by fairyarmadillo** [comment]
There is no leather in chitinous armor. It's all chitin. Leather is unnecessary as chitin IRL is not universally hard. Its natural texture is very soft, and it is naturally tanned via a process called sclerotization, but this doesn't happen uniformly. The chitin sclerotizes differently in different areas.. This gives the animal hard armor across broad sections of its body that becomes more flexible in places like the joints. A caterpillar's body is made of the same stuff as a lobster's.

Chitinous armor probably makes use of the softer more leathery bits where flexibility is needed as well as the hard parts where it is not.

**Comment by carlarctg** [comment]
Taking a break for 2 more days from digital stuff. Someone can take over this PR if they want to get it in faster.

File: data/json/items/armor/arms_armor.json [hunk 1]
```diff
@@ -160,9 +160,7 @@
     "price_postapoc": 1250,
     "name": { "str": "pair of biosilicified chitin arm guards", "str_pl": "pairs of biosilicified chitin arm guards" },
     "description": "A pair of arm guards crafted from the carefully cleaned and pruned biosilicified exoskeletons of acidic ants.  Acid-resistant but brittle.",
-    "material": [ "acidchitin" ],
-    "proportional": { "weight": 1.125, "volume": 1.13, "price": 1.25, "encumbrance": 1.5, "warmth": 1.5 },
-    "material_thickness": 4
+    "material": [ "acidchitin" ]
```
Comment (Maleclypse) [inline data/json/items/armor/arms_armor.json:163]:
```suggestion
    "material": [ "acidchitin" ],
```

File: data/json/items/armor/arms_armor.json [hunk 2]
```diff
@@ -161,8 +161,6 @@
     "name": { "str": "pair of biosilicified chitin arm guards", "str_pl": "pairs of biosilicified chitin arm guards" },
     "description": "A pair of arm guards crafted from the carefully cleaned and pruned biosilicified exoskeletons of acidic ants.  Acid-resistant but brittle.",
     "material": [ "acidchitin" ],
```
Comment (Maleclypse) [inline data/json/items/armor/arms_armor.json:163]:
```suggestion
    "material": [ "acidchitin" ]
```

**Comment by Jan-Blasiak** [comment]
Could we revisit this idea? Biochitin is still complete shit, but not making boots have 100% coverage makes it so you get annoying warnings every time you step on every bush in the forest.

The coverage currently means that part of the foot is completely exposed, which IRL boots of any kind shouldn't do. If material coverage is an issue then % coverage of chitin (slightly better) and simple leather (slightly worse resistance) should be used instead.

**Pull request merged**